src/hosted.js

```diff
@@ -95,7 +95,11 @@
     function load(Bramble) {
         Bramble.load("#bramble",{
             url: "index.html",
-            useLocationSearch: true
+            useLocationSearch: true,
+            enableCollaboration: true,
```

Comment:
useLocationSearch? Or enableCollaboration?
Since useLocationSearch does exist in master right now: 
https://github.com/mozilla/brackets/blob/master/src/hosted.js#L98